packages/opentelemetry-exporter-collector/src/index.ts

```diff
@@ -14,4 +14,4 @@
  * limitations under the License.
  */
 
-export * from './CollectorExporter';
+export * from './platform/node';
```

Comment:
it should be `export * from './platform';` otherwise the node version will be used  in web